### DISCUSSION TITLE
w3m-connect-button and auto-connect=true

### DISCUSSION THREAD

**Discussion by Orchius83** [WalletConnect/walletconnect-monorepo#2460] [Developer Support] [answered]
I want to be notified when the web3Modal auto reconnect for a page refresh. 
watchAccount and subscribeEvents are not triggered on the refresh of the page with the auto reconnect.
I'm using [html-wagmi](https://github.com/WalletConnect/web3modal-examples/tree/main/html-wagmi) example to test it

**Comment by glitch-txs** [comment, marked as answer]
What about [getAccount](https://wagmi.sh/core/actions/getAccount)?

**Reply by finessevanes** [reply]
You could also use [watchAccount](https://wagmi.sh/core/actions/watchAccount).
